CMakeLists.txt

```diff
@@ -25,12 +24,16 @@ if(HIGHFIVE_PARALLEL_HDF5)
     set(HDF5_PREFER_PARALLEL TRUE)
 endif()
 
-find_package(HDF5 QUIET REQUIRED)
+find_package(HDF5 REQUIRED)
 
 set(Boost_NO_BOOST_CMAKE TRUE)
 
 if(USE_BOOST)
-  find_package(Boost QUIET REQUIRED COMPONENTS system unit_test_framework)
+  set(BOOST_REQ_COMPONENTS system)
+  if(HIGHFIVE_UNIT_TESTS)
+    set(BOOST_REQ_COMPONENTS ${BOOST_REQ_COMPONENTS} unit_test_framework)
```

Comment:
What about using `list(APPEND ...)` instead of `set(BOOST_REQ_COMPONENTS ${BOOST_REQ_COMPONENTS} unit_test_framework)`?

Reply:
There is a strange case when it shows a `;` instead of being treated as 2+ values... Maybe when we use quotes?

Reply:
I just tried and it seems to work fine. Thanks for the hint, indeed it's neater